### PR TITLE
Fixed pom and added usage section in readme.

### PR DESCRIPTION
**Pull request by ghost**
Fixed encoding and made project version constant
due to Maven warnings.
Added usage instructions to readme file.

### CODE REVIEW

**Comment by ghost** [comment]
Please make a review @KPP13

File: pom.xml [hunk 1]
```diff
@@ -51,7 +51,7 @@
     </dependencies>
 
     <build>
-        <finalName>Robot</finalName>
+        <finalName>Maciek-robot</finalName>
```
Comment (KPP13) [inline pom.xml:54]: :+1: 
But, why no Tomcat-robot? :P

Comment (ghost) [reply pom.xml:54]: :ghost:

**Comment by KPP13** [comment]
Ok :) You can merge :) I tested it and I really like your app.

**Pull request merged**